<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDExecuterNlleOperation</class>
 <widget class="QDialog" name="BDExecuterNlleOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxDate">
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Date de l'opération</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_enLive">
        <property name="text">
         <string>En live</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_enDiffere">
        <property name="text">
         <string>En différé</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Date :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditDate">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="btn_calendrier">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Description de l'opération :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditDescription">
       <property name="maxLength">
        <number>160</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Paramétrage de l'opération</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidgetOperation">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>170</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>170</number>
        </attribute>
        <column>
         <property name="text">
          <string>N° compte débit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>N° compte crédit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Libellé du compte</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Montant débit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Montant crédit</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="btn_mvtDebiteur">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>23</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Débiter un compte</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_mvtCrediteur">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>23</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Créditer un compte</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_supprimerMvt">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_editer">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Editer</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_executer">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Exécuter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_annuler">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
